pymc/logprob/transforms.py

```diff
@@ -377,7 +393,15 @@ class MeasurableTransform(MeasurableElemwise):
 
     valid_scalar_types = (
         Exp,
+        Exp2,
+        Expm1,
+        Sigmoid,
         Log,
+        Log2,
+        Log10,
+        Log1p,
```

Comment:
The cases that don't require a new transform aren't needed here, since we convert them to previously supported cases